src/olympia/migrations/1005-disable-beta-files.sql

```diff
@@ -0,0 +1,6 @@
+UPDATE files
+SET status = 5
+WHERE id IN
```

Comment:
Why not just do `UPDATE files SET status = 5 WHERE status = 7` ?

Reply:
Because Friday brain...I'll update.